auto_tests/file_saving_test.c

```diff
@@ -83,9 +84,10 @@ static void load_data_decrypted(void)
     struct Tox_Options *options = tox_options_new(nullptr);
     ck_assert(options != nullptr);
 
+    tox_options_set_experimental_owned_data(options, true);
     tox_options_set_savedata_type(options, TOX_SAVEDATA_TYPE_TOX_SAVE);
-
-    tox_options_set_savedata_data(options, clear, size);
+    tox_options_set_savedata_data(options, clear, clear_size);
```

Comment:
Should this be `ck_assert`ed so there's no potential for a use-after-free in the test?